### PR TITLE
Composefix

### PR DESCRIPTION
**Pull request by runboj**
Modified the docker-compose.yaml file, line 27 to "./mongodb_data:/bitnami"

### CODE REVIEW

**Comment by dylanmcreynolds** [comment]
I'm not sure why this never popped up before...this has been like it was for three years. Did docker or podman change how to reference folders in the same folder? @minottic any ideas?

**Comment by minottic** [comment]
mmmm but that's a local volume, not a folder on the host. Also it should not be on the host, so no need to have it on the gitignore if I am not mistaken

**Comment by dylanmcreynolds** [comment]
But if it's not a folder on the host, the data won't persist, which I think we want for this, no?

**Comment by minottic** [comment]
in principle, the docker volume is persisted unless one removes it. Even if the containers stop, the volume should be kept. Mounting it on the host I guess would benefit from docker-deamon restarts though (I am not 100% sure if the daemon restart removes the volumes). This said I don't think we should aim for any failure-proof data persistence with this docker-compose which should just be a recipe to get started. 

In addition, wouldn't the host mounting create unnecessary files on the host? as well as risking to fall into the common permission and mounting docker problems? 

To summarise, I think there is no error in the docker-compose as is, I am not against mounting the folder on the host, but I think it should be a customisation every adopter is free to do, and shouldn't be committed here

**Comment by dylanmcreynolds** [comment]
@minottic , thanks. This whole issue arose when @runboj had errors trying to run a fresh scicatlive install. @runboj maybe you could revert the change locally and see, again, if it's necessary? I can't remember, were you running podman or docker?

**Comment by minottic** [comment]
can I close this?
